compiler/hash-reporting/src/render.rs

```diff
@@ -300,6 +313,31 @@ impl ReportCodeBlock {
                 " "
             };
 
+            // So if we're at the start of the 'skip' range, use '...' instead
+            if let Some(range) = skip_lines_range.clone() {
+                if *range.start() == index {
+                    let range_line_number = format!(
+                        "{:<pad_width$}",
+                        ".".repeat(3),
+                        pad_width = longest_indent_width + 2
```

Comment:
Why +2?

Reply:
For the `|` and the space between the line number and the separator.